api/handlers/combo.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+var winston = require('winston');
+
+var JoinEvent = require('../models/JoinEvent.js');
+var Transaction = require('../models/Transaction.js');
+
+var ComboHandler = {
+  feedItems: {
+    get: function(req, res) {
+      winston.log('info', 'feedItems.get called by ComboRouter');
+      Transaction.findPublic({}).then(function(transactions) {
+        JoinEvent.find().exec().then(function(evts) {
+          var combined = transactions.concat(evts);
+          combined.sort(function(a,b) {
+            return b.timeStamp - a.timeStamp;
+          });
+          return res.status(200).send(combined);
+        }, function(err) {
+          winston.log('error', 'Error finding transactions: %s', err);
```

Comment:
err.toString() or winston won't be able to print the err. Won't crash, but won't output the error